common/app/views/fragments/meta/contactAuthor.scala.html

```diff
@@ -4,24 +4,22 @@
 
 @tags.contributors.headOption.map { profile =>
     @profile.properties.twitterHandle.map{ twitter =>
-        <div class="meta__twitter">
+        <div class="meta__twitter meta__contact-method">
             <a href="https://twitter.com/@twitter" data-link-name="twitter-handle" data-component="meta-twitter-handle" class="@RenderClasses(Map(
-                    "button button--small button--secondary" -> (!amp && !ActiveExperiments.isParticipating(experiments.Garnett)),
-                    "rounded-icon social-icon centered-icon" -> amp
-                ), "tone-colour") ">
-                    @fragments.inlineSvg("twitter-bird", "icon", Seq("inline-tone-fill", "i-left"))
-                    <div class="contact hide-until-leftcol @if(amp) {u-h}">@@@twitter</div>
+                    "button button--small button--secondary" -> (!ActiveExperiments.isParticipating(experiments.Garnett)),
+                )) ">
+                    @fragments.inlineSvg("twitter-bird", "icon", Seq("i-left"))
+                    @@@twitter
```

Comment:
So this will now show on mobile all the time outside of AMP?

Reply:
Yep, it was made that way during Garnett so AMP matches normal articles now.